pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
 
 | Status                                       | Description                                                                                                                                                                      |
```

Comment:
I think we need to have a CTA that links to the blog post on this page - as that details each of the direct path changes e.g. online to operational. Perhaps detailing the date this changed

e.g. [date] We have recently made changes to the naming of device statuses. You can find out more details here.